assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -391,6 +394,26 @@ def get_model_signature(self):
             ),
         )
 
+    def _robust_load_config(self, config) -> transformers.PretrainedConfig:
+        """Check and modify if config has missing indices.
+
+        :param config: Config Object from transformers
+        :type config: PretrainedConfig
+        :return: Config Object
+        :rtype: PretrainedConfig
+        """
+        ids = list(config.id2label.keys())
```

Comment:
id2label will be present for all models? Could we try catch or check if id2label exist before proceeding?

Reply:
Good point Ayush. For saving it in transformers oss flavor, inference happens with id2label and its failing if the id2label is incorrectly formed. If it doesnt have id2label then the porting itself is not possible